src/app/admin/admin.module.ts

```diff
@@ -28,8 +28,8 @@ import { PaginationComponent } from './components/pagination/pagination.componen
 import { RelevancyDateComponent } from './components/relevancy-date/relevancy-date.component';
 import { MatDatepickerModule } from '@angular/material/datepicker';
 import { MatNativeDateModule } from '@angular/material/core';
-import { MatFormFieldModule } from '@angular/material/form-field';
-import { MatInputModule } from '@angular/material/input';
+import { MatLegacyFormFieldModule as MatFormFieldModule } from '@angular/material/legacy-form-field';
```

Comment:
Could you provide the reference to the Angular documentation about this?

Reply:
After some research i was able to find this explanation on [github](https://github.com/angular/components/blob/main/guides/v15-mdc-migration.md) (go to "What has changed?" section). 

My understanding is that these components have been deprecated, and can only be supported by the new version of angular if you use a legacy import.